components/applications-service/pkg/wrap/wrap_test.go

```diff
@@ -0,0 +1,51 @@
+package wrap
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/gogo/protobuf/proto"
+	"github.com/golang/protobuf/ptypes"
+	"github.com/stretchr/testify/require"
+
+	"github.com/chef/automate/api/external/applications"
+)
+
+func TestWrap(t *testing.T) {
+	ex1 := applications.ExampleOne{
+		IntOne: 23,
+		IntTwo: 42,
+	}
+
+	packedEx1, err := ptypes.MarshalAny(&ex1)
+	require.NoError(t, err)
+
+	wrap1 := applications.Wrapper{
+		Type:    "ExampleOne",
+		Message: packedEx1,
+	}
+
+	fmt.Printf("w: %+v\n", wrap1)
+
+	data, err := proto.Marshal(&wrap1)
+
+	require.NoError(t, err)
+
+	fmt.Printf("data: %+v\n", data)
+
+	var wrap1rt applications.Wrapper
+
+	err = proto.Unmarshal(data, &wrap1rt)
+
+	require.NoError(t, err)
+
+	fmt.Printf("w (rt): %+v\n", wrap1rt)
+
+	var extractedEx1 applications.ExampleOne
+
+	err = ptypes.UnmarshalAny(wrap1rt.Message, &extractedEx1)
```

Comment:
Another possibility (I think) is to unmarshal the any into an `interface{}` and do some coercion? But that seems not good.